src/video/win32_v.cpp

```diff
@@ -993,6 +993,23 @@ void VideoDriver_Win32Base::EditBoxLostFocus()
 	SetCandidatePos(this->main_wnd);
 }
 
+std::vector<int> VideoDriver_Win32Base::GetListOfMonitorRefreshRates() {
+	std::vector<int> rates = {};
+	EnumDisplayMonitors(nullptr, nullptr, [](HMONITOR hMonitor, HDC hDC, LPRECT rc, LPARAM data) -> BOOL {
+		auto& list = *reinterpret_cast<std::vector<int>*>(data);
```

Comment:
this is... horrifying. Since you're using a lambda anyway, why not just capture `rates` in the lambda and not use `data` at all?
`[&rates](HMONITOR...`